packages/babel-core/src/config/resolve-targets-browser.ts

```diff
@@ -17,10 +19,11 @@ export function resolveTargets(
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   root: string,
 ): Targets {
-  // todo(flow->ts) remove any and refactor to not assign different types into same variable
-  let targets: any = options.targets;
-  if (typeof targets === "string" || Array.isArray(targets)) {
-    targets = { browsers: targets };
+  const optTargets = options.targets;
+  let targets: InputTargets;
```

Comment:
`targets` will be `undefined` if `optTargets` is an object such as `{ chrome: 100 }`.